src/components/SideNav/_side-nav-carbon.scss

```diff
@@ -0,0 +1,147 @@
+@import '~carbon-components/scss/components/ui-shell/side-nav';
+
+/*****************************
+*
+* This file brings in Carbon sidenav styles and aligns the pattern changes
+* we have made in code. (ie. having buttons as well as anchors for sidenav
+* links, using isRail while still wanting collapse on small screens, etc. )
+*
+*****************************/
+
+// Disable for file since this is from Carbon
+
+/* stylelint-disable */
+
+.#{$iot-prefix}--side-nav {
+  ///////////////////////////////////////
+  // Added her to allow for collapse sidenav
+  // with isRail property functionality
+  ///////////////////////////////////////
+  @media (max-width: 65.99em) {
```

Comment:
Is there a variable or calculation for 65.99em? Seems oddly specific. At least we can put it in a variable here since it is used multiple times.

Reply:
It is the opposite of the variables breakpoints that Carbon has that are min-width. Applied here so I am not having to duplicate more styles. But I will make a variable